scripts/deposit-withdraw.ts

```diff
@@ -0,0 +1,103 @@
+import { JsonRpcProvider } from 'ethers/providers'
```

Comment:
These scripts are here just to demonstrate they work. They are mostly C/P from the Optimism repo. I'd plan to convert this ideally to a unit test, and expose a deployment function to use from G.